atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraClientFactory.java

```diff
@@ -203,10 +203,12 @@ public boolean validateObject(PooledObject<CassandraClient> client) {
 
     @Override
     public void destroyObject(PooledObject<CassandraClient> client) {
-        client.getObject().getOutputProtocol().getTransport().close();
-        log.debug("Closed transport for client {} of host {}",
-                UnsafeArg.of("client", client),
-                SafeArg.of("cassandraClient", CassandraLogHelper.host(addr)));
+        try (CassandraClient cassandraClient = client.getObject()) {
+            cassandraClient.close();
```

Comment:
Isn't `close()` getting called twice here, once explicitly and once due to the `try` block? Should we remove the `try` block and simply call `client.getObject().close()` directly?

Reply:
fixing, try-with-resources was leftover from before I pushed it into the `CassandraClient `